src/core/core-varlink.c

```diff
@@ -582,6 +561,42 @@ static int manager_varlink_init_user(Manager *m) {
         return 1;
 }
 
+int manager_setup_varlink_server(Manager *m, VarlinkServer **ret_s) {
+        _cleanup_(varlink_server_unrefp) VarlinkServer *s = NULL;
+        int r;
+
+        assert(m);
+        assert(ret_s);
+
+        if (m->varlink_server)
+                return 1;
```

Comment:
Hmm, what it the return convention, i.e. what is the difference between 0 and 1?

Reply:
Hmmm I'm not sure. It looks like this all used to return 0 but was changed in 064a5c1438690f9454ad9bff05ec4ec2acfa67d2. @DaanDeMeyer is there a reason why we return both 0 and 1?